packages/clay-badge/README.mdx

```diff
@@ -1,34 +1,39 @@
 ---
-title: 'Badges'
+title: 'Badge'
 description: 'Badges help highlight important information, such as notifications or new and unread messages.'
 lexiconDefinition: 'https://liferay.design/lexicon/core-components/badges/'
 packageNpm: '@clayui/badge'
-sibling: 'docs/components/css-badges.html'
+sibling: 'docs/components/css-badge.html'
 ---
 
 import {Badge} from '$packages/clay-badge/docs/index';
 
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
+-   [Composing](#composing)
 -   [API](#api)
 
 </div>
 </div>
 
-## Colors
+# Composing
 
-Badges can be used with the following colors:
-
-<Badge />
-
-We recommend that you review the use cases in the <a href="https://storybook-clayui.netlify.com/?path=/story/claybadge--default">Storybook</a>.
+Badges help highlight important information, such as notifications or new and unread messages.
 
 <div class="clay-site-alert alert alert-warning">
 	Badges are not used for non-numeric values. If you have a non-numeric value,
-	use labels instead.
+	use labels instead. Badges work for exact numbers up to 999. For numbers
```

Comment:
It's coming from Lexicon. I wanted to add more content so I just stole some from Lexicon in order to add some more content. Perhaps we need to make it clear what kind of information is contained on these pages.

1. Do we add any advice on how to best use the component?
2. Do we only have technical information on components (props, api)?
3. Do we have any of the content from Lexicon duplicated on Clay, or do we expect users to consult both before using the component?

Keeping in mind that our primary customers are people from DXP, does it hurt having some of the information from Lexicon duplicated on Clay? Sure, some person not attached to Liferay using Clay might get confused by this text, but then again, do we have people outside of Liferay visiting this site.

Reply:
> Perhaps we need to make it clear what kind of information is contained on these pages.

Yeah this is a great idea. Maybe we could do some research from other sites that that we think have good docs and condense it into some sort of structure for consistency. What would you think about opening a docs ticket where we can add some ideas of what we think would be best to include. These ideas would be similar to your three questions above. I am thinking something along the lines of

1. Overview/Introduction to component. Why/when/where do we use it
2. Variants (going from most used to least used)
3. Limitations/Pitfalls/
3. API Tables


We could do a first pass through and make it as simple as possible with demos, and then fill in areas we see as problematic or need more explaining. By doing this, it would give us a lens to view how we create succinct and consistent docs.

What do you think? If we establish this sort of gird before updating docs, it'll give us a great place to start updating docs.